spec/features/merchants/items/show_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe "Merchant Items Show Page" do
+
+
+  describe "Merchant Items Show Display", type: :feature do
+    before(:each) do 
+      @merchant_1 = create(:merchant)
+      @merchant_2 = create(:merchant)
+      @item_1 = create(:item, merchant: @merchant_1)
+      @item_2 = create(:item, merchant: @merchant_1) 
+      @item_3 = create(:item, merchant: @merchant_2) 
+    end
+
+    #User Story 7
```

Comment:
I would consider splitting up user story 7 testing across the show AND index spec files:
1) _When I click on the name of an item from the merchant items index page, (merchants/:merchant_id/items)
Then I am taken to that merchant's item's show page_. This first part of the user story should be written in `spec/features/merchants/items/index_spec.rb` -- basically just testing if the link you created on the index page does, in fact, take you to that specific show page (since you did change it from just a list to an actual link in this iteration).
2) _I am taken to that merchant's item's show page (/merchants/:merchant_id/items/:item_id)
And I see all of the item's attributes including: Name, Description, Current Selling Price._ This second part of the user story is what starts in the `spec/features/merchants/items/show_spec.rb` (this is basically what you have written in the show_spec currently. I would consider adding at least one to_not statements (like `expect(page).to_not have_content(@item_2.name)` to ensure the wrong item is not showing up on a show page.